extensions/amp-render/1.0/amp-render.js

```diff
@@ -335,6 +336,35 @@ export class AmpRender extends BaseElement {
       .then((html) => dict({'__html': html}));
   }
 
+  /**
+   *
```

Comment:
Can you please add a comment?

1. explaining why we do this
2. with a TODO to observe rewrapping at a lower level on `BaseTemplate`